<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferManagerItem</class>
 <widget class="QWidget" name="TransferManagerItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>48</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>784</width>
    <height>48</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>784</width>
    <height>48</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">TransferManagerItem
{
	background-color: transparent;
}
#pbTransfer, #pbTransferCompleted
{
	border: none;
}

#lTransferType , #lFileType, #lActionType, #lCancelTransfer, #lCompleted,
#lTransferTypeCompleted , #lFileTypeCompleted, #lActionTypeCompleted, #lCancelTransferCompleted, #lCompleted
{
	border: none;
}
#lTransferName, #lTransferNameCompleted
{
  font-family: &quot;Lato&quot;;
  font-size: 16px;
  font-weight: 400;
  color: #333333;
}
#lTotal, #lTotalCompleted
{
  font-family: &quot;Lato&quot;;
  font-size: 13px;
  font-weight: 400;
  color: #333333;
}
#lSpeed
{
  font-family: &quot;Lato&quot;;
  font-size: 13px;
  font-weight: 400;
  color: #aaaaaa;
}

#lRemainingTimeCompleted
{
   font: &quot;Lato&quot;;
   font-size: 10px;
   font-weight: normal;
   color: #333333;
}

#lRemainingTime
{
   font: &quot;Lato&quot;;
   font-size: 12px;
}

</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="sTransferState">
     <property name="minimumSize">
      <size>
       <width>784</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>784</width>
       <height>48</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="stateActive">
      <property name="minimumSize">
       <size>
        <width>784</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>784</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>25</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="wContainerLeft" native="true">
         <property name="minimumSize">
          <size>
           <width>608</width>
           <height>48</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>608</width>
           <height>48</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="minimumSize">
             <size>
              <width>608</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>608</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">padding-left: -2px;
padding-top: 1px;</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="lTransferType">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="lFileType">
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lTransferName">
               <property name="minimumSize">
                <size>
                 <width>275</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">padding-left: 0px;</string>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>10</width>
                 <height>16</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="lTotal">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">#lTotal
{
color:#777777;
text-decoration:none;
}</string>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lSpeed">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>195</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">padding-right: -2px;</string>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="pbTransfer">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>2</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>2</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="format">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wContainerRight" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lActionType">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../Resources_win.qrc">:/images/cloud_item_ico.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lRemainingTime">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="lCancelTransfer">
            <property name="minimumSize">
             <size>
              <width>12</width>
              <height>12</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>12</width>
              <height>12</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../Resources_win.qrc">
              <normaloff>:/images/clear_item_ico.png</normaloff>:/images/clear_item_ico.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>12</width>
              <height>12</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="stateCompleted">
      <property name="minimumSize">
       <size>
        <width>784</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>784</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>22</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="wContainerLeftCompleted" native="true">
         <property name="minimumSize">
          <size>
           <width>550</width>
           <height>38</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>550</width>
           <height>38</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widgetCompleted" native="true">
            <property name="minimumSize">
             <size>
              <width>550</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>550</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>1</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="lTransferTypeCompleted">
               <property name="minimumSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="lFileTypeCompleted">
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lTransferNameCompleted">
               <property name="minimumSize">
                <size>
                 <width>375</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>375</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">padding-bottom: 1px;</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>30</width>
                 <height>16</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="lCompleted">
               <property name="minimumSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lTotalCompleted">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">padding-left: 8px;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wContainerRightCompleted" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>9</number>
          </property>
          <property name="leftMargin">
           <number>13</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lActionTypeCompleted">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lRemainingTimeCompleted">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="lCancelTransferCompleted">
            <property name="minimumSize">
             <size>
              <width>12</width>
              <height>12</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>12</width>
              <height>12</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../Resources_win.qrc">
              <normaloff>:/images/clear_item_ico.png</normaloff>:/images/clear_item_ico.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>12</width>
              <height>12</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources_win.qrc"/>
 </resources>
 <connections/>
</ui>
